.github/issue_template.md

```diff
@@ -0,0 +1,10 @@
+---
+labels: needs-triage
+---
+
+<!--
+
+If you want to ask a general question, please create a new discussion instead of
+an issue: https://github.com/cometbft/cometbft/discussions
```

Comment:
I still thinking about if we should keep these refs or not. For cometbft related issues, referencing upstream definitely makes sense, but I'm not sure about all issues.